.github/workflows/ci-cd.yml

```
@@ -182,6 +182,16 @@ jobs:
           name: coverage-report
           path: htmlcov/
 
+  # ==========================================================================
+  # PR Fix Agent Integration
+  # ==========================================================================
+  pr-fix:
+    name: PR Fix Agent
+    needs: [test]
+    if: failure()
+    uses: ./.github/workflows/pr-fix-agent.yml
```

actionlint (1.7.10)

[error] error while parsing reusable workflow "./.github/workflows/pr-fix-agent.yml": yaml: line 112: could not find expected ':'

(workflow-call)